core/meta/module-object-descriptor.js

```diff
@@ -60,6 +60,9 @@ var ModuleObjectDescriptor = exports.ModuleObjectDescriptor = ObjectDescriptor.s
             this.super(serializer);
             this._setPropertyWithDefaults(serializer, "module", this.module);
             this._setPropertyWithDefaults(serializer, "exportName", this.exportName);
+            if(this.object) {
+                this._setPropertyWithDefaults(serializer, "object", this.object);
```

Comment:
@marchant Is there a spec for serializing/deserializing `ModuleObjectDescriptor#object`?